pkgs/string_scanner/CHANGELOG.md

```diff
@@ -0,0 +1,171 @@
+## 1.4.0
```

Comment:
We'll want to add a new changelog section and bump to as least 1.4.1-wip (if not just 1.4.1).